src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
     public class AzureAppConfigurationOptions
     {
         private const int MaxRetries = 2;
-        private static readonly TimeSpan MaxRetryDelay = TimeSpan.FromMinutes(1);
+        internal static readonly TimeSpan MaxRetryDelay = TimeSpan.FromMinutes(1);
```

Comment:
Is this change necessary?

Reply:
Referenced it in `StartupOptions` to try and keep this value consistent for both files, but I could just use a constant there too.

Reply:
Do we want to support a different MaxRetryDelay for startup vs refresh?
@jimmyca15

Reply:
Looks like my second comment got swallowed. Although I expect we would use the same time, I don't think this should be shared in that manner. A separate field to reference where needed seems appropriate.